metadata_backend/services/metax_mapper.py

```diff
@@ -235,11 +235,15 @@ def __init__(self, object_type: str, metax_data: Dict, data: Dict) -> None:
         self.research_dataset = metax_data
         self.datacite_data = data
         self.affiliations: List = []
-        with open(METAX_REFERENCE_FILE, "r", encoding="utf-8") as ref_file:
-            metax_reference_data = json.load(ref_file)
-        self.identifier_types = metax_reference_data["identifier_types"]
-        self.languages = metax_reference_data["languages"]
-        self.fields_of_science = metax_reference_data["fields_of_science"]
+        for file in METAX_REFERENCE_FILES:
+            with open(file, "r", encoding="utf-8") as ref_file:
```

Comment:
This whole block is looking not so ideal. I don't like the check if there is a key in the JSON object. I'd rather `METAX_REFERENCE_FILES` was a dictionary for this kind of implementation.

But when thinking a bit more about it, I'd rather have the file opening and loading files into memory in `conf.py`. What do you think?

Reply:
Good point